test/e2e/utils.ts

```diff
@@ -0,0 +1,22 @@
+import os from "os";
```

Comment:
It's so weird that GH does not syntax highlight these files as typescript? This one doesn't even have the `.test.ts`, so there's really no reason for it not to 🤔